src/main/java/innovimax/mixthem/operation/ICharOperation.java

```diff
@@ -1,15 +1,17 @@
 
 package innovimax.mixthem.operation;
+
 import innovimax.mixthem.MixException;
 
+import java.util.stream.IntStream;
+
 interface ICharOperation extends IOperation {
 	/**
-	* Returns the result of the operation (maybe empty).
+	* Returns the result of the operation as an int stream.
 	* @param c1 The first character to zip (maybe -1)
 	* @param c2 The second character to zip (maybe -1)
-	* @return The result of the operation (maybe empty)
-	* @throws MixException - If an mixing error occurs
-	* @throws ProcessException - If process must be ended
+	* @return The result of the operation as an int stream
+	* @throws MixException - If an mixing error occurs	
 	*/	
-	int[] process(int c1, int c2) throws MixException, ProcessException;
+	IntStream process(int c1, int c2) throws MixException;
```

Comment:
why is the signature here different from ILineOperation.process(line1, line2, result) ?

Reply:
this will be done

Reply:
when ?